docs/local.md

```diff
@@ -0,0 +1,266 @@
+```
+
+### Register Resources
+
+```bash
+# Register fungible resource ID with erc20 contract
+cb-sol-cli bridge register-resource --resourceId "0x000000000000000000000000000000c76ebe4a02bbc34786d860b355f5a5ce00" --targetContract "0x21605f71845f372A9ed84253d2D024B7B10999f4"
+
+# Register non-fungible resource ID with erc721 contract
+cb-sol-cli bridge register-resource --resourceId "0x000000000000000000000000000000e389d61c11e5fe32ec1735b3cd38c69501" --targetContract "0xd7E33e1bbf65dC001A0Eb1552613106CD7e40C31" --handler "0x3f709398808af36ADBA86ACC617FeB7F5B7B193E"
+
+# Register generic resource ID
+cb-sol-cli bridge register-generic-resource --resourceId "0x000000000000000000000000000000f44be64d2de895454c3467021928e55e01" --targetContract "0xc279648CE5cAa25B9bA753dAb0Dfef44A069BaF4" --handler "0x2B6Ab4b880A45a07d83Cf4d664Df4Ab85705Bc07" --hash --deposit "" --execute "store(bytes32)"
+```
+
+### Specify Token Semantics
+
+To allow for a variety of use cases, the ethereum contracts support both the `transfer` and the `mint/burn` ERC methods.
+
+To simplify these examples we should use `mint/burn`:
+
+```bash
+# Register the erc20 contract as mintable/burnable
+cb-sol-cli bridge set-burn --tokenContract "0x21605f71845f372A9ed84253d2D024B7B10999f4"
+
+# Register the associated handler as a minter
+cb-sol-cli erc20 add-minter --minter "0x3167776db165D8eA0f51790CA2bbf44Db5105ADF"
+
+# Register the erc721 contract as mintable/burnable
+cb-sol-cli bridge set-burn --tokenContract "0xd7E33e1bbf65dC001A0Eb1552613106CD7e40C31" --handler "0x3f709398808af36ADBA86ACC617FeB7F5B7B193E"
+
+# Add the handler as a minter
+cb-sol-cli erc721 add-minter --minter "0x3f709398808af36ADBA86ACC617FeB7F5B7B193E"
+```
+
+## On-Chain Setup (Substrate)
+
+### Add Relayer
+
+We need to register the account of the relayer on substrate (cb-sol-cli deploys contracts with the 5 test keys preloaded). 
+
+Select the `Sudo` tab in the PolkadotJS UI. Choose the `addRelayer` method of `chainBridge`, and select Alice as the relayer.
+
+### Register Resources
+
+Select the `Sudo` tab and call `chainBridge.setResourceId` for each of the transfer types you wish to use:
+
+**Fungible (Native asset):**
+
+Id: `0x000000000000000000000000000000c76ebe4a02bbc34786d860b355f5a5ce00`
+
+Method: `0x4578616d706c652e7472616e73666572` (utf-8 encoding of "Example.transfer")
+
+**NonFungible(ERC721):**
+
+Id: `0x000000000000000000000000000000e389d61c11e5fe32ec1735b3cd38c69501`
+
+Method: `0x4578616d706c652e6d696e745f657263373231` (utf-8 encoding of "Example.mint_erc721")
+
+**Generic (Hash Transfer):**
+
+Id: `0x000000000000000000000000000000f44be64d2de895454c3467021928e55e01`
+
+Method:  `0x4578616d706c652e72656d61726b` (utf-8 encoding of "Example.remark")
+
+### Whitelist Chains
+
+Using the `Sudo` tab, call `chainBridge.whitelistChain`, specifying `0` for out ethereum chain ID.
+
+## Running A Relayer
+
+Here is an example config file for a single relayer using the contracts we deployed.

```

Comment:
```suggestion
Here is an example config file for a single relayer ("Alice") using the contracts we've deployed.

```

I think it might make sense to reference alice here, makes the next line more clear